app/models/helpers.py

```diff
@@ -0,0 +1,15 @@
+from flask import abort, make_response
+from .planet import Planet
+
+def validate_planet(id):
+	try:
+		id = int(id)
+	except:
+		return abort(make_response({"message": f"planet {id} is invalid"}, 400))
+
+	planet = Planet.query.get(id)
+
+	if not planet:
+		abort(make_response({"message": f"planet {id} not found"}, 404))
+
+	return planet
```

Comment:
Helpers look great! 👍  We may want to consider making the validation code as generic as possible to process any class and id or renaming this file to `helper_planet_routes` or `planet_routes_helper`. Either would be helpful if we expand this project to contain more routes for `moons`, `suns`, etc.

Reply:
A more generic validation helper method would look like:

```python
from flask import make_response, abort

def validate_object(cls, id):
    try:
        id = int(id)
    except:
        return abort(make_response({"message": f"{cls} {id} is invalid"}, 400))

    obj = cls.query.get(id)
    
    if not obj:
        abort(make_response({"message": f"{cls} {id} not found"},404))
    
    return obj
```